engine/src/test/java/com/datatorrent/stram/StramMiniClusterTest.java

```diff
@@ -227,6 +242,7 @@ public void testSetupShutdown() throws Exception
   private LogicalPlan createDAG(LogicalPlanConfiguration lpc) throws Exception
   {
     LogicalPlan dag = new LogicalPlan();
+    addExtraTestJar(dag);
```

Comment:
There will be duplicate of code.. Because most of the things I want to do are part of testSetupShutdown test.
But I get your point, I'll have a seperate test than the current one to test this functionality.